djog/admin.py

```diff
@@ -1,3 +1,4 @@
 from django.contrib import admin
-
+from djog.dogsModels.dogs import Breeds, Dogs
```

Comment:
I think we need to call folder 'model' - its default name. Your name could cause misunderstanding.